src/snapred/backend/service/CalibrationService.py

```diff
@@ -374,11 +388,102 @@ def assessQuality(self, request: CalibrationAssessmentRequest):
 
         return record
 
+    @FromString
+    def normalization(self, request: NormalizationCalibrationRequest):
+        groupingFile = request.groupingPath
+        groupingScheme = groupingFile.split("/")[-1].split(".")[0].replace("SNAPFocGroup_", "")
```

Comment:
at this point this split split replace should live in a method.

Reply:
I'm hoping when the grouping index is completed, it will not longer be necessary to do this.